types/operator_metadata.go

```diff
@@ -52,8 +52,8 @@ func (om *OperatorMetadata) Validate() error {
 		return errors.New("logo is required")
 	}
 
-	if !isImageURL(om.Logo) {
-		return errors.New("logo must be a valid image url")
+	if err := isImageURL(om.Logo); err != nil {
+		return errors.New("logo must be a valid image url with err: " + err.Error())
```

Comment:
I would just return the err here instead of adding more stuff, since unclear if it actually adds anything, plus it could also be moved down to the error in `isImageURL`.

Reply:
yes. fixed